src/main/java/com/spotinst/sdkjava/model/bl/gcp/OceanGKEInstanceTypes.java

```diff
@@ -0,0 +1,63 @@
+package com.spotinst.sdkjava.model.bl.gcp;
```

Comment:
Do we need this class? InstanceTypes are just a list of strings, that can be represented fully in the LauncSpecification class itself.